expertise/execute_expertise.py

```diff
@@ -0,0 +1,177 @@
+from pathlib import Path
+import openreview
+from .create_dataset import OpenReviewExpertise
+from .dataset import ArchivesDataset, SubmissionsDataset, BidsDataset
+from .config import ModelConfig
+
+# Move run.py functionality to a function that accepts a config dict
+def execute_expertise(config_file=None, model_config=None):
+
+    if model_config:
+        config = model_config
+    elif config_file:
+        config = ModelConfig(config_dict=config_file)
```

Comment:
What if you always expect `config_file` and create the ModelConfig object here instead of using if else statements?

Reply:
Ah, that does sound like a better solution - I'll make that change

Reply:
What if the only parameter is ModelConfig and it is created before calling execute_expertise?